src/content/docs/features/email.md

```diff
@@ -2,11 +2,17 @@
 title: Email Handlers
 ---
 
-Membrane programs are capable of sending and receiving emails. Recieving an email works similar to how [HTTP endpoints](/features/endpoints/) are handled. All you need to do is export an `email` function from the root of your program and use the program's unique email address.
+Membrane programs are capable of sending and receiving emails. Recieving an email works similarly to how [HTTP endpoints](/features/endpoints/) are handled. All you need to do is export an `email` function from the root of your program and use the program's unique email address.
+
+<!-- TODO: create reference section for `sms` and link to that instead -->
+
+:::note
+Membrane also supports sms! To send yourself a text programmatically, add the `sms` program—installed in your workspace by default—as a connection and call `nodes.sms.send({ message: "hi from Membrane" })`.
+:::
 
 ## Basic Example
 
-This program receives an email and uses the `email` program to forward the email to the currently signed in user's inbox. In order to run this program make sure you [add `email` as a dependency](/features/dependencies/#add-a-program-dependency)
+This program receives an email and uses the `email` program to forward the email to the currently signed in user's inbox. In order to run this program make sure you [add `email` as a dependency](/features/connections/#add-a-program-connection).
```

Comment:
Are we calling dependencies _connections_ as a Membrane concept? The `/features/connections` section does that, but under concepts we have the subheading as _dependencies_

Reply:
This is something Juan and I were discussing. The biggest issue we have is distinguishing an NPM dependency vs a membrane dependency. In the editor we're currently calling connections to other nodes/programs _dependencies_, but I'm leaning towards just calling them `connections`. That's how you would describe to attached nodes in a graph and I think it does a decent job describing what's happening in an understandable way. 

Curious to hear your input though. Is that terminology weird?

Reply:
I've created T-692 to track updating gaze with this language.